packages/babel-plugin-proposal-private-methods/test/fixtures/private-method/reassignment/output.js

```diff
@@ -0,0 +1,16 @@
+var counter = 0;
+
+var _privateMethod = new WeakSet();
+
+class Foo {
+  constructor() {
+    _privateMethod.add(this);
+
+    babelHelpers.classPrivateMethodSet(++counter);
```

Comment:
Since `classPrivateMethodSet` doesn't take any parameter, I'd prefer to transform this to `++counter, babelHelpers.classPrivateMethodSet()`.

Reply:
Actually I thought about that for a while, but the sequence expression comes at 1 byte more.

Reply:
It’s 2 bytes if you include the space.